2024-02-01 19:39:54 jdbc[14]: null
org.h2.message.DbException: Połączenie zostało zamknięte przez aplikacje i zostało usunięte przez kolektor nieużytków
The connection was not closed by the application and is garbage collected [90018-214]
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:190)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:132)
	at org.h2.Driver.connect(Driver.java:59)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138)
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:359)
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:201)
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:470)
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:733)
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:712)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1623)
Caused by: org.h2.jdbc.JdbcSQLNonTransientConnectionException: Połączenie zostało zamknięte przez aplikacje i zostało usunięte przez kolektor nieużytków
The connection was not closed by the application and is garbage collected [90018-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	... 16 more
2024-02-01 20:01:04 jdbc[4]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-214]
2024-02-01 20:12:57 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-214]
2024-02-01 20:18:40 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-214]
2024-02-01 20:20:07 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-214]
2024-02-01 20:30:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-214]
2024-02-01 20:31:28 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-214]
2024-02-01 20:33:38 jdbc[4]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-214]
2024-02-01 20:43:38 jdbc[4]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-214]
2024-02-01 20:44:19 jdbc[6]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-214]
2024-02-01 20:45:41 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-214]
2024-02-01 20:46:21 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-214]
2024-02-02 00:44:03 jdbc[4]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-214]
2024-02-02 00:44:47 jdbc[4]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-214]
2024-02-02 00:45:09 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-214]
2024-02-02 00:46:34 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-214]
2024-02-02 00:47:25 jdbc[4]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Obiekt jest zamknięty
The object is already closed [90007-214]
